backend/src/sms/middlewares/sms-transactional.middleware.ts

```diff
@@ -91,9 +93,11 @@ async function sendMessage(
       action,
     })
 
-    const BAD_REQUEST_ERRORS = [TemplateError, InvalidRecipientError]
-    if (BAD_REQUEST_ERRORS.some((errType) => err instanceof errType)) {
-      res.status(400).json({ message: (err as Error).message })
+    if (err instanceof InvalidPhoneNumberError) {
+      res.status(400).json({
+        code: 'invalid_phone_number',
```

Comment:
I think maybe make it a bit more generic `invalid_recipient`? so we can use the same code for other endpoints too